compiler/src/iree/compiler/Codegen/SPIRV/Passes.cpp

```diff
@@ -118,7 +118,8 @@ static void addBufferizePasses(OpPassManager &passManager,
 static void addTileAndDistributeToWorkgroupsPasses(
     OpPassManager &passManager, bool useFuseTensorPadWithConsumerPass = false,
     bool useWARForCooperativeMatrixCodegen = false) {
-  passManager.addPass(createTileAndDistributeToWorkgroupsPass());
+  passManager.addPass(
+      createTileAndDistributeToWorkgroupsPass(kNumMaxParallelDims));
```

Comment:
why is this needed? This is already the default value?

Reply:
I removed them all